spec/features/admin_can_approve_stores_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.feature "platform admin can approve a pending store" do
+  scenario "can approve pending store" do
+    user = User.create(username: "bacon", password: "password")
```

Comment:
I think there should be a user and an admin. The user creates a store and then the admin separately approves or suspends the store